### DISCUSSION TITLE
Multiple fake accounts with repos that contains only viruses

### DISCUSSION THREAD

**Discussion by renwixx** [community/community#123794] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hi guys! Yesterday I found many accounts that contain only viruses, like info stealer Lumma. I can't send too many reports, so I want to publish their profiles here. Maybe, someone from GitHub staff take notice of all these accounts and ban them. Thanks.

https://github.com/Boutions/
https://github.com/KeeleySmith1/
https://github.com/LeticiaCornett/
https://github.com/MakeMeCool/
https://github.com/MichaelWadri/
https://github.com/Rajasekar155/
https://github.com/RibeiroY/
https://github.com/SKATEBOARDVR/
https://github.com/attationin/
https://github.com/denkisadie834/
https://github.com/fikeselena68/
https://github.com/guytervad/
https://github.com/he-cosmo11/
https://github.com/headless71/
https://github.com/hediye80//
https://github.com/higorzica/
https://github.com/ketamiune/
https://github.com/pointanbyloka927/
https://github.com/prometeytron/
https://github.com/servisonly/
https://github.com/svenundefinedshire/
https://github.com/tfl1971/
https://github.com/toluzane/
https://github.com/webmoney-wiki/
https://github.com/yori21/

There guys are selling the viruses:
https://github.com/crackers-services/
https://github.com/Awakennn1/
https://github.com/wisemenservices/

**Comment by ExpiredWater** [comment]
You could always report the uses, on the left hand side under achievements.

![image](https://github.com/community/community/assets/24321664/0f8179b1-bd1e-4a90-8b5d-732b8d31a0be)

**Reply by renwixx** [reply]
I know that. But I have limitations, for example, I can only send 2 or 3 reports, and then I have to wait for the results of checking those reports... It takes too much time

**Comment by ebndev** [comment]
Hi @MrFlooky, 👋🏻 We really appreciate you flagging this. 

In the future, the best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
